network/network.go

```diff
@@ -103,6 +103,8 @@ type Stat struct {
 	Direction Direction
 	// Opened is the timestamp when this connection was opened.
 	Opened time.Time
+	// Transient indicates that this connection is transient and may be closed soon
```

Comment:
```suggestion
	// Transient indicates that this connection is transient and may be closed soon.
```

Reply:
sigh....

Reply:
@vyzo 

You just have to click on accept man. All sentences above this one have the period :)